_posts/2018-02-13-integrate-and-deploy-react-with-phoenix.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: Integrate and Deploy React with Phoenix
+date: 2018-02-19 09:00:00 -04:00
+categories: post
+desc: End-to-end guide on introducing a React frontend to a Phoenix application.
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+> You've just finished your lightning fast Phoenix JSON API, what's next?
+
+## Motivation
+
+My most recent side project, [Contact](https://www.github.com/mhanberg/contact), is a JSON REST API written in Phoenix, designed to be the backend to an instant messaging application (e.g., Slack). There was a hackathon coming up at work, and I thought it'd be fun to make a frontend for Contact during it, and although Contact's development was thoroughly test-driven, I wanted to make sure that my API was ready to be used. 
```

Comment:
I'm probably not the target audience for this post, but I would have appreciated a hyperlink to the phoenix website. It's not easy to google.

Reply:
(or maybe just a short definition of what the Phoenix Framework is and why I'd want to us it on a thing)

Reply:
Good point, thanks!